src/test/lazy-omap-stats/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ add_executable(ceph_test_lazy_omap_stats
   main.cc
   lazy_omap_stats_test.cc)
 target_link_libraries(ceph_test_lazy_omap_stats
-  librados ${UNITTEST_LIBS} Boost::system)
+  librados ceph-common ${UNITTEST_LIBS} Boost::system)
```

Comment:
The last commit update brings this change. It fixes a FTBFS that popped up on one of my environments. The problem was about unresolved symbol: `_ZN4ceph6buffer7v15_2_03ptr7releaseEv`. It think it leaked to the `librados` client because `buffer::~ptr()`, which is the caller of `ceph::buffer::v15_2_0::ptr::release()`, is defined in `buffer.h` so a compiler may inline it.